docusaurus/docs/ui-extensions/register-an-extension.md

```diff
@@ -0,0 +1,128 @@
+---
+id: register-an-extension
+title: Register an extension from your app plugin
+sidebar_label: Register an extension
+sidebar_position: 1
+description: Learn how to register an extension from your app plugin.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - links
+  - extensions
+  - app plugins
+---
+
+## Before you begin
+
+Be sure your plugin meets the following requirements before proceeding:
+
+- It must be an app plugin.
+- It must be preloaded by setting the [preload property](https://grafana.com/docs/grafana/latest/developers/plugins/metadata/#pluginjson) to `true` in the `plugin.json`.
+- It must be installed and *enabled*.
```

Comment:
```suggestion
- It must be installed and *enabled* for the extension to be available.
```

Its tricky as this doc is about creating the extension logic in the first place, but this point in particular is more about "your extensions will only be activated if this is true".

Reply:
I thought it would be good to link to someone on enabling app plugins, but I think we don't actually document this anywhere - I certainly can't find it. Maybe something for us to follow up on in https://grafana.com/docs/grafana/latest/administration/plugin-management/ @josmperez